very easy/10.js

```diff
@@ -1,5 +1,5 @@
 function divisibleByFive(n) {
-
+    return n % 5 === 0 ? true : false;

```

Comment:
jeśli zwracasz true/false w ternary to możesz je całkowicie pomijać
```return n % 5 === 0```
Da taki sam efekt